src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -174,7 +174,14 @@ var DOMPropertyOperations = {
         if (namespace) {
           node.setAttributeNS(namespace, attributeName, '' + value);
         } else {
-          node.setAttribute(attributeName, '' + value);
+          if (propertyInfo.hasBooleanValue ||
```

Comment:
Can make the above an `else if` and then the `else` stays the same (and we avoid nested conditional logic)